frontend/src/employee-frontend/components/vasu/VasuEditPage.tsx

```diff
@@ -145,12 +146,18 @@ export default React.memo(function VasuEditPage({
             {showSpinner && <Spinner />}
           </StatusContainer>
           {vasu && (
-            <Button
-              text={i18n.vasu.checkInPreview}
-              disabled={status.state != 'clean'}
-              onClick={() => history.push(`/vasu/${vasu.id}`)}
-              primary
-            />
+            <ButtonContainer>
+              <Button
+                text={i18n.vasu.checkInPreview}
+                disabled={status.state != 'clean'}
+                onClick={() => history.push(`/vasu/${vasu.id}`)}
+                primary
+              />
+              <Button
+                text={i18n.vasu.leavePage}
+                onClick={() => history.goBack()}
+              />
```

Comment:
"Poistu" is a little bit confusing term here.

1. Open vasu from child page
2. Click Edit
3. Click check in preview
4. Click Edit
5. Click Leave page 4 times to actually leave the page.

Reply:
Changed the target of the button to always go back to the child information page.